src/commands/team.rs

```diff
@@ -118,11 +118,11 @@ pub async fn run(ctx: Context, command: ApplicationCommandInteraction) {
 
                 if all_prs[0..2].len() == index {
                     pull_req_cloned +=
-                        &format!("<:reply:1029065416905076808>[{}]({})\n", pr.title, pr.url);
```

Comment:
seems like you replaced the url fields on team.rs as well, this uses a different api
This file is probably better left untouched for issues related to heroes :smile: .
removing the changes from this file should probably fix the failing test